src/synthesizer/extensions/particle_spectra_with_shift.c

```diff
@@ -42,10 +42,10 @@ int find_nearest_bin(double lambda, double *grid_wavelengths, int nlam) {
  * @param spectra: The output array.
  */
 static void shifted_spectra_loop_cic_serial(struct grid *grid, struct particles *parts,
-                                    double *spectra) {
+                                    double *spectra, c) {
```

Comment:
Needs to be `const double c` because its a floating point number we want precision on and we know it won't ever be changed.